scripts/RT/weps2sweep_worker.py

```
@@ -0,0 +1,367 @@
+    """
+    # We should be fully within a thread at this point...
+    try:
+        # Parse and validate the payload using Pydantic model
+        job = WEPS2SweepJobPayload.model_validate_json(payload)
+        run_weps(job)
+
+    except ValidationError as exp:
+        # Invalid payload structure - log the validation errors
+        LOG.error("Invalid payload format: %s", exp)
+        LOG.error("Raw payload: %s", payload[:200])  # Log first 200 chars
+    except Exception as exp:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)